WCFDataService/Client/System/Data/Services/Client/Materialization/HttpWebResponseMessage.cs

```diff
@@ -87,14 +88,15 @@ public HttpWebResponseMessage(HttpWebResponse httpResponse)
         /// <param name="headers">The headers.</param>
         /// <param name="statusCode">The status code.</param>
         /// <param name="getResponseStream">A function returning the response stream.</param>
-        internal HttpWebResponseMessage(HeaderCollection headers, int statusCode, Func<Stream> getResponseStream)
+        internal HttpWebResponseMessage(HeaderCollection headers, int statusCode, Func<Stream> getResponseStream, IODataResponseMessage underlyingResponseMessage)
```

Comment:
getResponseStream and  underlyingResponseMessage are kind of dup. Why not add an  overloaded contructor to accept  underlyingResponseMessage only, so other caller does not need change.